modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -22,18 +22,16 @@ definition {
 		}
 	}
 
-	tearDown {
-		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
-
-		ObjectAdmin.deleteAllCustomObjectsViaAPI();
-
-		if (${testLiferayVirtualInstance} == "true") {
-			PortalInstances.tearDownCP();
-		}
-		else {
-			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
-		}
-	}
+	//tearDown {
```

Comment:
Hey @juliosilveira77 , this code should be a comment?

case should please remove this block. lets avoid push comment